packages/nvidia_nat_test/src/nat/test/utils.py

```
@@ -89,6 +91,69 @@ async def run_workflow(*,
+    response_payload = {}
+    try:
+        deadline = time.time() + 60  # 60 second timeout waiting for the workflow to respond
+        response = None
+        while response is None and time.time() < deadline:
+            try:
+                response = requests.post(url=f"{workflow_url}/generate",
+                                         json={"messages": [{
+                                             "role": "user", "content": question
+                                         }]},
+                                         timeout=60)
+            except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)